{"name": "User can search for product -- @1.3 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome=129.0.6668.60); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF6C833FDA5+29557]\n\t(No symbol) [0x00007FF6C82B2240]\n\t(No symbol) [0x00007FF6C816B6EA]\n\t(No symbol) [0x00007FF6C81722BE]\n\t(No symbol) [0x00007FF6C8174637]\n\t(No symbol) [0x00007FF6C81746F0]\n\t(No symbol) [0x00007FF6C81B95AB]\n\t(No symbol) [0x00007FF6C81E74FA]\n\t(No symbol) [0x00007FF6C81B3006]\n\t(No symbol) [0x00007FF6C81E7710]\n\t(No symbol) [0x00007FF6C82088A3]\n\t(No symbol) [0x00007FF6C81E72A3]\n\t(No symbol) [0x00007FF6C81B12DF]\n\t(No symbol) [0x00007FF6C81B2451]\n\tGetHandleVerifier [0x00007FF6C866DCBD+3363469]\n\tGetHandleVerifier [0x00007FF6C86B9B47+3674391]\n\tGetHandleVerifier [0x00007FF6C86AEAEB+3629243]\n\tGetHandleVerifier [0x00007FF6C83FFC66+815670]\n\t(No symbol) [0x00007FF6C82BD6EF]\n\t(No symbol) [0x00007FF6C82B92B4]\n\t(No symbol) [0x00007FF6C82B9450]\n\t(No symbol) [0x00007FF6C82A81FF]\n\tBaseThreadInitThunk [0x00007FFBED717374+20]\n\tRtlUserThreadStart [0x00007FFBED85CC91+33]\n\n", "trace": "  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\search_results_steps.py\", line 46, in verify_search_results\n    context.app.search_results_page.verify_search_results(product)\n  File \"C:\\Users\\Owner\\python-selenium-automation\\pages\\search_results_page.py\", line 74, in verify_search_results\n    self.verify_partial_text(expected_product, *self.SEARCH_RESULTS_TXT)\n  File \"C:\\Users\\Owner\\python-selenium-automation\\pages\\base_page.py\", line 92, in verify_partial_text\n    actual_text = self.driver.find_element(*locator).text\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 90, in text\n    return self._execute(Command.GET_ELEMENT_TEXT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Open target main page", "status": "passed", "start": 1727372104595, "stop": 1727372108106}, {"name": "When Search for iphone", "status": "passed", "start": 1727372108106, "stop": 1727372114341}, {"name": "Then Verify search results show for iphone", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome=129.0.6668.60); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF6C833FDA5+29557]\n\t(No symbol) [0x00007FF6C82B2240]\n\t(No symbol) [0x00007FF6C816B6EA]\n\t(No symbol) [0x00007FF6C81722BE]\n\t(No symbol) [0x00007FF6C8174637]\n\t(No symbol) [0x00007FF6C81746F0]\n\t(No symbol) [0x00007FF6C81B95AB]\n\t(No symbol) [0x00007FF6C81E74FA]\n\t(No symbol) [0x00007FF6C81B3006]\n\t(No symbol) [0x00007FF6C81E7710]\n\t(No symbol) [0x00007FF6C82088A3]\n\t(No symbol) [0x00007FF6C81E72A3]\n\t(No symbol) [0x00007FF6C81B12DF]\n\t(No symbol) [0x00007FF6C81B2451]\n\tGetHandleVerifier [0x00007FF6C866DCBD+3363469]\n\tGetHandleVerifier [0x00007FF6C86B9B47+3674391]\n\tGetHandleVerifier [0x00007FF6C86AEAEB+3629243]\n\tGetHandleVerifier [0x00007FF6C83FFC66+815670]\n\t(No symbol) [0x00007FF6C82BD6EF]\n\t(No symbol) [0x00007FF6C82B92B4]\n\t(No symbol) [0x00007FF6C82B9450]\n\t(No symbol) [0x00007FF6C82A81FF]\n\tBaseThreadInitThunk [0x00007FFBED717374+20]\n\tRtlUserThreadStart [0x00007FFBED85CC91+33]\n\n", "trace": "  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\search_results_steps.py\", line 46, in verify_search_results\n    context.app.search_results_page.verify_search_results(product)\n  File \"C:\\Users\\Owner\\python-selenium-automation\\pages\\search_results_page.py\", line 74, in verify_search_results\n    self.verify_partial_text(expected_product, *self.SEARCH_RESULTS_TXT)\n  File \"C:\\Users\\Owner\\python-selenium-automation\\pages\\base_page.py\", line 92, in verify_partial_text\n    actual_text = self.driver.find_element(*locator).text\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 90, in text\n    return self._execute(Command.GET_ELEMENT_TEXT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Owner\\python-selenium-automation\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1727372114341, "stop": 1727372115791}, {"name": "Then Verify correct search results url opens for iphone", "status": "skipped", "start": 1727372130207, "stop": 1727372130207}], "parameters": [{"name": "product", "value": "iphone"}, {"name": "expected_result", "value": "iphone"}], "start": 1727372100148, "stop": 1727372130207, "uuid": "fca1b00c-9a72-44ba-947c-a9beef9e64bf", "historyId": "87d04be6a21e3ab181f68ed27af4b900", "fullName": "Target main page search tests: User can search for product", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Target main page search tests"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}